utils/similarity/encode.py

```diff
@@ -0,0 +1,188 @@
+import os
+import sys
+
+from slither import Slither
+from slither.slithir.operations import *
```

Comment:
Consider using explicit import

Reply:
There is a lot of things to import: https://github.com/crytic/slither/blob/dev-simil/slither/slithir/operations/__init__.py 

:sob: